### PR TITLE
Commit

### PR DESCRIPTION
**Pull request by crishellco**
## Ticket
* n/a

## Changes
* Add commit view
* Merge in new icons
* Merge in dex.js, assets.js
* Fix some language references

## Notes
`services/dex.js` is a direct copy of the desktop wallet code, so there are definitely certain things not up to our current code standards. Resolving those should be considered tech debt that we will need to address.

## Screenshots
image
image
image

## Code Coverage
n/a

### CODE REVIEW

**Comment by jsolman** [comment]
Issues:
1. Blocks until eligible for fees is showing a negative number instead of showing how many blocks until eligible to compound after eligible to commit. Also if already eligible to compound it should just say it is eligible to compound.
2. Compound button is able to be clicked even when it is grayed out
3. Max button on commit dialog doesn't work

Asside from those issues it looks to be working reasonably well.

File: src/components/AuthenticatedWrapper.vue [hunk 1]
```diff
@@ -135,24 +143,35 @@ export default {
       flex: 1;
       display: flex;
       flex-direction: column;
-      justify-content: space-between;
-      margin: $space-lg 0 $space-xl;
+      // justify-content: center;
```
Comment (schaeferCode) [inline src/components/AuthenticatedWrapper.vue]: Is this left for some reason?

File: src/components/AuthenticatedWrapper.vue [hunk 2]
```diff
@@ -135,24 +143,35 @@ export default {
...
+          &.commit {
+            height: toRem(34px);
           }
 
           &.wallet {
-            height: toRem(38px);
+            height: toRem(30px);
+          }
+
+          &.ico {
```
Comment (schaeferCode) [inline src/components/AuthenticatedWrapper.vue:183]: Should this be `.icon`?

File: src/components/Commit.vue [hunk 3]
```diff
@@ -0,0 +1,678 @@
...
+            transform-origin: left center;
+            transform: scale(.8);
+          }
+        }
+      }
+    }
+  }
+}
+</style>
+
```
Comment (schaeferCode) [inline src/components/Commit.vue]: Remove extra lines.

File: src/main.js [hunk 4]
```diff
@@ -96,9 +96,12 @@ const language = localStorage.getItem('language') ||
 
 // Create VueI18n instance with options
 const i18n = new VueI18n({
-  locale: language,
   fallbackLocale: 'en',
+  locale: language,
   messages,
+  missing() {
+    //
```
Comment (schaeferCode) [inline src/main.js]: This is coming later?

**Review by schaeferCode — APPROVED** [review]
Looks good so far. Just a few minor comments. I'm going to pull the branch down and mess with it some too. 

NOTES: 
I just pulled down the PR. I get a ton of warnings about `SOME_WORD` is not a string. I think these have to do with the translate tool we're using. I'm assuming we'll fill those in later? 

screen shot 2018-09-19 at 8 42 10 pm

There's no way to cancel or exit this modal once it pops up. Can easily be fixed in a later PR though. 

Everything looks great though!

File: src/components/AuthenticatedWrapper.vue (hunk 2 above)
Comment (crishellco) [reply src/components/AuthenticatedWrapper.vue:183]: Nope

**Comment by crishellco** [comment]
@schaeferCode 
screen shot 2018-09-20 at 1 33 48 am

File: src/l10n/en.json [hunk 5]
```diff
@@ -1,133 +1,302 @@
...
+  "thisWillBeAMakerOrderLeftOnTheBook": "This will be a maker order that will be left on the book until someone takes it. You will not be charged a fee for this trade.",
+  "time": "Time",
+  "to": "To",
+  "toDownload": "to download",
+  "toReceiveATotalOf": "to receive a total of",
+  "toReceiveAnEstimatedTotalOf": "to receive an estimated total of",
+  "token": "Token",
+  "tokenSaleSuccessful": "Token Sale Participation Successful. Your Balance of {symbol} is now {balance}",
   "tokenStats": "Token Stats",
-  "totalBalance": "Total Balance",
```
Comment (jsolman) [inline src/l10n/en.json]: Why is this removed? This is used.
Need to be verify careful with merging over translation strings.

Comment (jsolman) [reply src/l10n/en.json]: Please audit to make sure other removed strings aren't being used.

**Review by jsolman — CHANGES_REQUESTED** [review]

**Review by jsolman — APPROVED** [review]

**Pull request merged**